### ISSUE TITLE
Unable to run bundler/r10k

### ISSUE THREAD

**Issue by nikhilo** [rfkrocktk/docker-puppetserver#4]
The container needs following packages in order to `bundle install`
- git
- make
- gcc

Some sensible default for GEM_PATH will make the container usable out of the box

Also, I saw following error while running r10k

```
bash-4.2$ r10k deploy environment -p
/usr/share/rubygems/rubygems/dependency.rb:296:in `to_specs': Could not find 'r10k' (>= 0) among 3 total gem(s) (Gem::LoadError)
```

I had seen similar error `Could not find 'bundler'` with bundler before. I fixed it with `gem install r10k`
I think these are due to GEM_PATH not being set. Because after I installed r10k it's complaining about not finding json gem.

**Comment by naftulikay** [comment]
All of these issues appear related to the absence of a home directory for the Puppet user. I'm working on fixes right now.

**Comment by naftulikay** [comment]
Fixed in v1.1.3-3.
